src/pages/files/Upload.tsx

```diff
@@ -83,9 +83,9 @@ export function Upload(): ReactElement {
         // The website is in some directory, remove it
         if (idx.commonPrefix) {
           const substrStart = idx.commonPrefix.length
-          indexDocument = idx.indexPath.substr(substrStart)
-          fls = files.map(f => {
-            const path = (f.path as string).substr(substrStart)
+          indexDocument = idx.indexPath.slice(substrStart)
+          fls = fls.map(f => {
```

Comment:
```suggestion
          fls = files.map(f => {
```
This would reintroduce bug #365